### PR TITLE
add career guide post

### PR DESCRIPTION
**Pull request by anajsana**
This commit adds career guide post

### CODE REVIEW

File: content/en/guides/career-development.md [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+## Integrating the unique with the standard
+
+Organizational variances are common even though patterns in how most companies think about and manage open source are similar. For example, all companies must decide how to handle compliance with open source software distribution, code contributions and code releases. There are also security, talent recruitment, developer relations, and other functions that companies must assign to a team to ensure all goes well. 
+
+"There's all these different things that every company deals with. But the subset of those things that happen to fall within the open source office is what's different at every company," said Norris. Google’s OSPO headed by Will Norris is an example of a team focused on engineering and compliance. A separate team handles the program side at Google.
```
Comment (choeppler) [inline content/en/guides/career-development.md:96]: In line 46 Will Norris is introduced as open source lead at Twitter. And here the text reads as if he's simultaneously heading Google's OSPO which may be confusing...

Comment (anajsana) [reply content/en/guides/career-development.md]: Right! I've made a commit to fix this. Thank you for taking the time to review this! @choeppler

**Comment by anajsana** [comment]
hey, @caniszczyk I think this guide is done :+1: Is it possible to merge it today? It would be great to add it as an OSPO new in the upcoming OSPOlogy meeting

File: content/en/guides/career-development.md [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+**Table of Contents**
+
+- [Guide Description](#guide-description)
+- [Defining the scope of work for OSPO roles](#defining-the-scope-of-work-for-ospo-roles)
+- [Refitting existing career ladders for open source](#refitting-existing-career-ladders-for-open-source)
+- [Integrating the unique with the standard](#integrating-the-unique-with-the-standard)
+- [Mapping Skill Categories](#mapping-skill-categories)
+- [Example: Mapping the Skills of a Community Manager](#example-mapping-the-skills-of-a-community-manager)
+- [Retaining and Promoting Your OSPO Professionals: Job Ladders](#retaining-and-promoting-your-OSPO-professionals-job-ladders)
```
Comment (xmulligan) [inline content/en/guides/career-development.md:17]:
```suggestion
- [Retaining and Promoting Your OSPO Professionals: Job Ladders](#retaining-and-promoting-your-ospo-professionals-job-ladders)
```

Comment (anajsana) [reply content/en/guides/career-development.md]: Thank you!

**Comment by caniszczyk** [comment]
@anajsana LGTM, can you draft a blog post for this to go out later this week, like Thursday?

**Comment by anajsana** [comment]
> @anajsana LGTM, can you draft a blog post for this to go out later this week, like Thursday?

Sure, working on it!

**Comment by anajsana** [comment]
@caniszczyk FYI: https://github.com/todogroup/todogroup.org/pull/229

**Pull request merged**